### PR TITLE
Feature/internal api network

### PR DESCRIPTION
**Pull request by JJGothard**
Hi @preethammavin & @SueSmith ! Since there's such a tight turn-around to get this doc out on the day of release ( "The Day of Release" -- new band name, I called it! 🤪) I wanted to start getting the content reviewed ASAP. I'm currently working on pulling new screenshots from stage.

Thanks!
Jacque

### CODE REVIEW

**Comment by JJGothard** [comment]
Now that I'm seeing what's on stage, @preethammavin , it looks like there will be changes to the content/flow. I'm working on changing that right now as well.

**Comment by JJGothard** [comment]
@SueSmith , this got the thumbs up from @preethammavin  (he said he left one comment, but I'm not seeing it here...) so I'm converting this from "draft" to "ready for review." Thanks! 🙏

**Comment by JJGothard** [comment]
Also, all the of the screenshots and the flows are consistent with what's currently on stage 😊🤪🙀

File: src/pages/docs/postman/collaboration/adding-private-network.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+---
+title: "Adding to the Private API Network"
```
Comment (SueSmith) [inline src/pages/docs/postman/collaboration/adding-private-network.md:2]: Given that this covers both adding and using maybe change the title to "Your private API network" or something? it isn't a verb phrase but nbd, if you can think of one that isn't too long great.. 😅 (no need to change the file name or index, if you just adjust the title text here that's perfect)

I'd lose the capitalization as per the slack convo around branded terms

File: src/pages/docs/postman/collaboration/adding-private-network.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+order: 66
```
Comment (SueSmith) [inline src/pages/docs/postman/collaboration/adding-private-network.md:3]: another page is 66 already (audit logs), you can use e.g. 66.1 or 59.9 to make it after or before

File: src/pages/docs/postman/collaboration/adding-private-network.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    name: "Next Steps"
+  - type: link
+    name: "Authoring your documentation"
+    url: "/docs/postman/api-documentation/authoring-your-documentation/"
+
+---
+
+You can publish the APIs that your teams use internally to the Private API Network. When you publish to the Private API Network, your team can access the APIs they need and start consuming them immediately.
+
+![Team APIs](https://assets.postman.com/postman-docs/Internal+Network+Explore.jpg)
```
Comment (SueSmith) [inline src/pages/docs/postman/collaboration/adding-private-network.md:29]: the pic seems to be public apis?

File: src/pages/docs/postman/collaboration/adding-private-network.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+    url: "https://www.youtube.com/watch?v=8tLvvQ-3Nx0"
+  - type: section
+    name: "Next Steps"
+  - type: link
+    name: "Authoring your documentation"
+    url: "/docs/postman/api-documentation/authoring-your-documentation/"
+
+---
+
+You can publish the APIs that your teams use internally to the Private API Network. When you publish to the Private API Network, your team can access the APIs they need and start consuming them immediately.
```
Comment (SueSmith) [inline src/pages/docs/postman/collaboration/adding-private-network.md:27]: again I'd lose the capitalization

File: src/pages/docs/postman/collaboration/adding-private-network.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+![Team APIs](https://assets.postman.com/postman-docs/Internal+Network+Explore.jpg)
+
+You can add to the Private API Network from your team's private API space on the [Dashboard](https://www.postman.com). The [Dashboard](https://www.postman.com) is also where your team members can discover and consume your private APIs.
```
Comment (SueSmith) [inline src/pages/docs/postman/collaboration/adding-private-network.md:31]: if you can use the dashboard url we talked about on that other pr (app.getpostman.com)

File: src/pages/docs/postman/collaboration/adding-private-network.md (hunk 4 above)
Comment (SueSmith) [reply src/pages/docs/postman/collaboration/adding-private-network.md:27]: I won't comment on all of these but if you can have a pass through the doc for them..

File: src/pages/docs/postman/collaboration/adding-private-network.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+    * [Security](#security)
+    * [Guidelines](#guidelines)
+* [Discovering and consuming private APIs](#discovering-and-consuming-private-apis)
+* [Next steps](#next-steps)
+
+### Adding your APIs
+
+> Before you can add an API to the Private API Network, all of your team members must be given permission to view the API. Learn more about [roles and permissions](/docs/postman/collaboration/roles-and-permissions/).
+
+To add a private API for your team, sign in to your [Dashboard](https://www.postman.com) and click **API Network** at the top.
```
Comment (SueSmith) [inline src/pages/docs/postman/collaboration/adding-private-network.md:45]: if you can update all dashboard links to app.getpostman.com

File: src/pages/docs/postman/collaboration/adding-private-network.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+
+> Open your browser to sign in to the [Dashboard](https://www.postman.com).
+
+![Nav bar](https://assets.postman.com/postman-docs/Network+in+nav+bar.jpg)
+
+By default, your team's private API space will open. Click **Add new** on the right.
+
+![Team name](https://assets.postman.com/postman-docs/Add+new+button.jpg)
+
+In the modal, choose the workspace the API is coming from, and select the API that you want to publish. Click **Add to Network**.
```
Comment (SueSmith) [inline src/pages/docs/postman/collaboration/adding-private-network.md:55]: I'd maybe reword the first sentence to be a wee bit more specific here, "the workspace containing the collection you want to publish to the network" or something along those lines

File: src/pages/docs/postman/collaboration/adding-private-network.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+
+> APIs that have already been added to the Private API Network cannot be added again. If a team member tries to add an API that already exists in the network, they will receive an error message telling them that the API has already been added.
+
+#### Security
+
+When users sign in to Postman, only the private APIs for the teams associated with their login credentials will be visible to them. Users must be signed in to browse private APIs. Private APIs will not be discoverable or accessible to anyone who is not a part of your team.
+
+#### Guidelines
+
+To keep your APIs secure, you will have to make sure that every member of your team has viewing permission for the API you want to add. APIs submitted to the Private API Network without all team members having view-access will not be premitted to add. Learn more about team [roles and permissions](/docs/postman/collaboration/roles-and-permissions/).
```
Comment (SueSmith) [inline src/pages/docs/postman/collaboration/adding-private-network.md:79]: think I'd lose the hyphen in "view-access"

Comment (SueSmith) [reply src/pages/docs/postman/collaboration/adding-private-network.md:79]: typo "premitted"

File: src/pages/docs/postman/collaboration/adding-private-network.md [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+
+By default, your team's private API space will open, and you can browse through the list of your organization's private APIs.
+
+![API list](https://assets.postman.com/postman-docs/Private+API+listings.jpg)
+
+Click on an API to see details.
+
+![API display](https://assets.postman.com/postman-docs/Private+API+display.jpg)
+
+If you have edit access to the API, you can make chages directly from the [Dashboard](https://www.postman.com) view.
```
Comment (SueSmith) [inline src/pages/docs/postman/collaboration/adding-private-network.md:103]: typo "chages"

File: src/pages/docs/postman/collaboration/adding-private-network.md [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+
+#### Security
+
+When users sign in to Postman, only the private APIs for the teams associated with their login credentials will be visible to them. Users must be signed in to browse private APIs. Private APIs will not be discoverable or accessible to anyone who is not a part of your team.
+
+#### Guidelines
+
+To keep your APIs secure, you will have to make sure that every member of your team has viewing permission for the API you want to add. APIs submitted to the Private API Network without all team members having view-access will not be premitted to add. Learn more about team [roles and permissions](/docs/postman/collaboration/roles-and-permissions/).
+
+> View-access is the minimum permission you need to grant your teammates to be abe to add to the network. You can choose to give members of your team edit-access.
```
Comment (SueSmith) [inline src/pages/docs/postman/collaboration/adding-private-network.md:81]: typo "abe"

Comment (SueSmith) [reply src/pages/docs/postman/collaboration/adding-private-network.md:81]: again I'd lose the hyphens before "access"

File: src/pages/docs/postman/collaboration/adding-private-network.md [hunk 11]
```diff
@@ -0,0 +1,123 @@
...
+
+> Open your browser to sign in to the [Dashboard](https://www.postman.com).
+
+![Nav bar](https://assets.postman.com/postman-docs/Network+in+nav+bar.jpg)
+
+By default, your team's private API space will open, and you can browse through the list of your organization's private APIs.
+
+![API list](https://assets.postman.com/postman-docs/Private+API+listings.jpg)
+
+Click on an API to see details.
```
Comment (SueSmith) [inline src/pages/docs/postman/collaboration/adding-private-network.md:99]: "click on" > "click"

Comment (SueSmith) [reply src/pages/docs/postman/collaboration/adding-private-network.md:99]: or "select" is fine too

File: src/pages/docs/postman/collaboration/adding-private-network.md [hunk 12]
```diff
@@ -0,0 +1,123 @@
...
+
+To generate a collection, click **Generate collection** in the upper right.
+
+![Edit API](https://assets.postman.com/postman-docs/Generate+collection+button.jpg)
+
+Name the collection and select your reason for creating the collection. Click **Generate Collection**.
+
+![Generation modal](https://assets.postman.com/postman-docs/Collection+generation+modal.jpg)
+
+To view the collection, navigate to your team workspace and click **Collections**. The collection will be displayed in a list of all the collections in that workspace.
```
Comment (SueSmith) [inline src/pages/docs/postman/collaboration/adding-private-network.md:117]: maybe just a wee rewording on last sentence here something like "The collection will appear in the list alongside all other collections in the workspace." (just to remove passive..)

File: src/pages/docs/postman/collaboration/adding-private-network.md (hunk 1 above)
Comment (JJGothard) [reply src/pages/docs/postman/collaboration/adding-private-network.md:2]: Done :) Kept "Private API Network" in caps; my thinking was the branding discussion we had on Slack...

File: src/pages/docs/postman/collaboration/adding-private-network.md (hunk 2 above)
Comment (JJGothard) [reply src/pages/docs/postman/collaboration/adding-private-network.md:3]: Done.

File: src/pages/docs/postman/collaboration/adding-private-network.md (hunk 1 above)
Comment (JJGothard) [reply src/pages/docs/postman/collaboration/adding-private-network.md:2]: JK: removed caps

File: src/pages/docs/postman/collaboration/adding-private-network.md (hunk 3 above)
Comment (JJGothard) [reply src/pages/docs/postman/collaboration/adding-private-network.md:29]: yea, sorry, that was a beta pic/flow left-over. Fixed.

File: src/pages/docs/postman/collaboration/adding-private-network.md (hunk 5 above)
Comment (JJGothard) [reply src/pages/docs/postman/collaboration/adding-private-network.md]: Per our Slack discussion, I used the go.postman.co that @preethammavin suggested.

File: src/pages/docs/postman/collaboration/adding-private-network.md (hunk 6 above)
Comment (JJGothard) [reply src/pages/docs/postman/collaboration/adding-private-network.md]: Done as above

File: src/pages/docs/postman/collaboration/adding-private-network.md (hunk 7 above)
Comment (JJGothard) [reply src/pages/docs/postman/collaboration/adding-private-network.md:55]: So with this, the api doesn't have to be in collection form to share to the workspace. There also don't have to be docs. You can add just the schema you build. What about "the workspace in which you built the API"? Or "the workspace where the API was built"? "the workspace containing the API you want to publish to the network"? 

I went with the last one.

File: src/pages/docs/postman/collaboration/adding-private-network.md (hunk 8 above)
Comment (JJGothard) [reply src/pages/docs/postman/collaboration/adding-private-network.md:79]: Done & done!

File: src/pages/docs/postman/collaboration/adding-private-network.md (hunk 10 above)
Comment (JJGothard) [reply src/pages/docs/postman/collaboration/adding-private-network.md]: Done and done

File: src/pages/docs/postman/collaboration/adding-private-network.md (hunk 9 above)
Comment (JJGothard) [reply src/pages/docs/postman/collaboration/adding-private-network.md]: Done

File: src/pages/docs/postman/collaboration/adding-private-network.md (hunk 12 above)
Comment (JJGothard) [reply src/pages/docs/postman/collaboration/adding-private-network.md:117]: Done

**Comment by JJGothard** [comment]
@SueSmith , all feedback has been incorporated. Thanks! 👍

File: src/pages/docs/postman/collaboration/adding-private-network.md (hunk 7 above)
Comment (SueSmith) [reply src/pages/docs/postman/collaboration/adding-private-network.md]: ooh interesting, didn't know it worked like this

File: src/pages/docs/postman/collaboration/adding-private-network.md [hunk 13]
```diff
@@ -0,0 +1,123 @@
...
+    name: "Authoring your documentation"
+    url: "/docs/postman/api-documentation/authoring-your-documentation/"
+
+---
+
+You can publish the APIs that your teams use internally to the private API network. When you publish to the private API network, your team can access the APIs they need and start consuming them immediately.
+
+![Team APIs](https://assets.postman.com/postman-docs/Private+APIs.jpg)
+
+You can add to the private API network from your team's private API space on the [Dashboard](go.postman.co). The [Dashboard](go.postman.co) is also where your team members can discover and consume your private APIs.
```
Comment (SueSmith) [inline src/pages/docs/postman/collaboration/adding-private-network.md:31]: the links need to have https on them so `https://go.postman.co`

File: src/pages/docs/postman/collaboration/adding-private-network.md [hunk 14]
```diff
@@ -0,0 +1,123 @@
...
+
+* [Adding your APIs](#adding-your-apis)
+    * [Security](#security)
+    * [Guidelines](#guidelines)
+* [Discovering and consuming private APIs](#discovering-and-consuming-private-apis)
+* [Next steps](#next-steps)
+
+### Adding your APIs
+
+> Before you can add an API to the private API network, all of your team members must be given permission to view the API. Learn more about [roles and permissions](/docs/postman/collaboration/roles-and-permissions/).
```
Comment (SueSmith) [inline src/pages/docs/postman/collaboration/adding-private-network.md:43]: nbd for the release but if you could reword the first sentence here to make it more active (either here or in a follow up pr after release) e.g. "You can only add an API to the network if all team members have permission to view the API."

File: src/pages/docs/postman/collaboration/adding-private-network.md [hunk 15]
```diff
@@ -0,0 +1,123 @@
...
+
+> APIs that have already been added to the private API network cannot be added again. If a team member tries to add an API that already exists in the network, they will receive an error message telling them that the API has already been added.
+
+#### Security
+
+When users sign in to Postman, only the private APIs for the teams associated with their login credentials will be visible to them. Users must be signed in to browse private APIs. Private APIs will not be discoverable or accessible to anyone who is not a part of your team.
+
+#### Guidelines
+
+To keep your APIs secure, you will have to make sure that every member of your team has viewing permission for the API you want to add. APIs submitted to the private API network without all team members having view access will not be permitted to add. Learn more about team [roles and permissions](/docs/postman/collaboration/roles-and-permissions/).
```
Comment (SueSmith) [inline src/pages/docs/postman/collaboration/adding-private-network.md:79]: similar request here for the second sentence, if we can reword to make it more active e.g. "You cannot add an API to the network unless all team members have view access to it."

File: src/pages/docs/postman/collaboration/adding-private-network.md [hunk 16]
```diff
@@ -0,0 +1,123 @@
...
+
+By default, your team's private API space will open, and you can browse through the list of your organization's private APIs.
+
+![API list](https://assets.postman.com/postman-docs/Private+API+listings.jpg)
+
+Select an API to see details.
+
+![API display](https://assets.postman.com/postman-docs/Private+API+display.jpg)
+
+If you have edit access to the API, you can make chages directly from the [Dashboard](go.postman.co) view.
```
Comment (SueSmith) [inline src/pages/docs/postman/collaboration/adding-private-network.md:103]: still seeing "chages" here

File: src/pages/docs/postman/collaboration/collaboration-intro.md [hunk 17]
```diff
@@ -14,6 +14,11 @@ contextual_links:
   - type: link
     name: "Scaling Team Collaboration at Atlassian"
     url : https://www.youtube.com/watch?v=wYBZrdukack
+  - type: subtitle
+    name: "Documentation"
```
Comment (SueSmith) [inline src/pages/docs/postman/collaboration/collaboration-intro.md:18]: we don't normally use "documentation" as a section, I would maybe put this in a next steps section instead..?

File: src/pages/docs/postman/collaboration/adding-private-network.md (hunk 13 above)
Comment (JJGothard) [reply src/pages/docs/postman/collaboration/adding-private-network.md:31]: Done.

File: src/pages/docs/postman/collaboration/adding-private-network.md (hunk 14 above)
Comment (JJGothard) [reply src/pages/docs/postman/collaboration/adding-private-network.md:43]: Done

File: src/pages/docs/postman/collaboration/adding-private-network.md (hunk 15 above)
Comment (JJGothard) [reply src/pages/docs/postman/collaboration/adding-private-network.md:79]: Done

File: src/pages/docs/postman/collaboration/adding-private-network.md (hunk 16 above)
Comment (JJGothard) [reply src/pages/docs/postman/collaboration/adding-private-network.md:103]: Done.

File: src/pages/docs/postman/collaboration/collaboration-intro.md (hunk 17 above)
Comment (JJGothard) [reply src/pages/docs/postman/collaboration/collaboration-intro.md:18]: Done

**Comment by JJGothard** [comment]
@SueSmith , all feedback has been incorporated. Thanks! 👍
